packages/app/cypress/e2e/cypress-in-cypress-run-mode.cy.ts

```diff
@@ -0,0 +1,71 @@
+describe('Cypress In Cypress - run mode', { viewportWidth: 1200 }, () => {
+  it('e2e run mode spec runner header is correct', () => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.findBrowsers()
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer()
+    cy.visitApp()
+
+    simulateRunModeInUI()
+
+    cy.contains('dom-content.spec').click()
+    cy.location().should((location) => {
+      expect(location.hash).to.contain('dom-content.spec')
+    })
+
+    cy.findByTestId('aut-url').should('be.visible')
+    cy.findByTestId('playground-activator').should('not.exist')
+
+    // confirm expected content is rendered
+    cy.contains('1000x660').should('be.visible')
+    cy.contains('84%').should('be.visible')
+    cy.contains('Chrome 1').should('be.visible')
+    cy.contains('http://localhost:4455/cypress/e2e/dom-content.html').should('be.visible')
+
+    cy.percySnapshot()
```

Comment:
I know the styles in these snapshots aren't complete, but even as-is you'll run into non-determinant UI with the reporter duration:
![Screen Shot 2022-02-15 at 1 48 42 PM](https://user-images.githubusercontent.com/1711637/154137927-f365ce02-a8cb-48a6-9b60-53a08917c7b4.png)

Though I don't even see the unstyled header markup in that snapshot: https://percy.io/cypress-io/cypress/builds/15753852/changed/889019696?browser=chrome&browser_ids=20&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1200&widths=250%2C300%2C400%2C500%2C600%2C800%2C900%2C1000%2C1032%2C1200%2C1280

Reply:
We should consider removing these snapshots until [UNIFY-1150](https://cypress-io.atlassian.net/browse/UNIFY-1150) can be completed.

Reply:
I wonder if it's better to leave them in. They can be approved for now, and when the CSS issue is fixed, these would show up again in the diff automatically and get reviewed. If we make some other way to automatically force these back in (maybe skip for a week based on Date.now()?) I'm ok with that too.

Reply:
We have a pattern for this in `reporter` already, we customize the Percy command to remove non-deterministic elements: https://github.com/cypress-io/cypress/blob/73e936e28992d5f533a8341a923dd61719bd12d5/packages/reporter/cypress/support/e2e.ts#L9-L12

Reply:
It's also possible to send elementOverrides on a per-snapshot basis, but you have to work around the typing:
```ts
  // @ts-ignore
  cy.percySnapshot({
    elementOverrides: {
      '.runnable-header .duration': ($el) => {
        $el.text('XX:XX')
      },
    },
  })
```

But these and the existing snapshots are going to need a lot of work once UNIFY-1150 completes. So long as we're not tripping over them in Percy until that work lands, I don't have a strong opinion whether they stay or go. If they do go, we should log an issue to make sure they're not lost.